src/mkdocs_bibtex/utils.py

```diff
@@ -145,8 +214,29 @@ def insert_citation_keys(citation_quads, markdown):
 
     grouped_quads = [list(g) for _, g in groupby(citation_quads, key=lambda x: x[0])]
     for quad_group in grouped_quads:
-        full_citation = quad_group[0][0]  # the first key in the whole citation
+        full_citation = quad_group[0][0]  # the full citation block
         replacement_citaton = "".join(["[^{}]".format(quad[2]) for quad in quad_group])
+
+        # if cite_inline is true, convert full_citation with pandoc and add to replacement_citaton
+        if csl and bib:
+            try:
+                # Verify that the pandoc installation is newer than 2.11
+                pandoc_version = pypandoc.get_pandoc_version()
+                pandoc_version_tuple = tuple(int(ver) for ver in pandoc_version.split("."))
+                if pandoc_version_tuple <= (2, 11):
+                    raise Exception(f"Your version of pandoc (v{pandoc_version}) is incompatible with "
```

Comment:
Raise a runtime error instead of a blanket exception here
```suggestion
                    raise RuntimeError(f"Your version of pandoc (v{pandoc_version}) is incompatible with "
```